ui/dashboards/src/context/DashboardProvider/panel-group-slice.ts

```diff
@@ -57,17 +63,24 @@ export type PanelGroupId = number;
 
 export interface PanelGroupDefinition {
   id: PanelGroupId;
-  items: GridItemDefinition[];
   isCollapsed: boolean;
   title?: string;
+  itemLayouts: PanelGroupLayout[];
+  itemPanelKeys: Record<PanelGroupLayoutId, string>;
 }
 
+export interface PanelGroupLayout extends Layout {
```

Comment:
can we rename this to PanelGroupItemLayout? PanelGroupLayout makes it seems like its the layout for the whole panel group itself

Reply:
naming is hard 😆

Reply:
Yup, good call.